AndroidManifest.xml

```diff
@@ -150,16 +134,6 @@
                     android:mimeType="application/octet-stream"
                     android:scheme="file" />
             </intent-filter>
-
-            <!-- qq -->
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-                <action android:name="android.intent.action.EDIT" />
-                <category android:name="android.intent.category.DEFAULT" />
-                <data android:scheme="file"/>
-                <data
-                    android:pathPattern=".*\.osz" />
-            </intent-filter>
```

Comment:
Is there a reason why this was removed?

Reply:
It was accidental. I was trying to get implicit intents to work, and erroneously removed that and didn't put it in again after testing. I'll put it back in